doc/topics/releases/2016.11.9.rst

```diff
@@ -1297,14 +1297,14 @@ Changelog for v2016.11.8..v2016.11.9
 
 * **ISSUE** `#43223`_: (`rallytime`_) Properly deprecate describe_route_table function in boto_vpc module (refs: `#43445`_)
 
-* **PR** `#43456`_: (`rallytime`_) Add Neon to version list
+* **PR** `#43456`_: (`rallytime`_) Add 3000 to version list
```

Comment:
we probably shouldn't update commit messages as those should be set to what they were previously